first_meeting.md

```diff
@@ -0,0 +1,272 @@
+# TODO
+
+- http://jsbin.com/yubuso/1/edit?html,css
+- Embed Something
+  - Twitter
+  	- https://about.twitter.com/resources/buttons#tweet
+  - Youtube
+  	- <iframe width="560" height="315" src="https://www.youtube.com/embed/_UoMXF73j0c" frameborder="0" allowfullscreen></iframe>
+  - SoundCloud
+  	- Embed
+  - Instagram
+  	- http://snapwidget.com/#getstarted
+  - Pinterest
+  - Facebook Share ~~Button~~
+
+
+
+# First Meeting
+
+## Overview
+
+The primary objective of the first meeting is to get everyone to come back
+to the second meeting.
+
+You can do this by
+
+- [continuing to be extremely welcoming](#continuing-to-be-extremely-welcoming)
+- [having every hacker ship something on their first day](#experiencing-success-on-the-first-day)
+
+### Continuing to be extremely welcoming
+
+Like at the club fair, continue
+[being extremely welcoming](/club_fair.md#be-extremely-welcoming).
+
+Treat everyone like they are the most important guest. Be overly hospitable.
+
+Why? Because if a newcomer feels like they feel
+
+- welcome in the club
+- that they have become friends with people in the club
+
+Then
+
+- they will come back to club
+- club will be much more fun
+
+```
+Pro-tip!
+
+{o,o}
+./)_)
+  " "
+
+Make sure to go over *explicitly* how to be hospitable with EVERY officer and
+EVERY old member. Know that most people WILL NOT do this by default.
+
+Don't fall into the trap that it is not worth doing or thinking that people will
+do this automatically.
+```
+
+### Experiencing success on the first day
+
+Everyone should ship something on their first day at Hack Club. It's the job of
+anyone else at the club who's been there at least once to help any newcommers
+ship that website.
+
+This being the first day of club this year, everyone in the entire club
+should ship a new website, even if they shipped one previously.
+
+**Why is it so important to ship something on someone's first meeting?**
+
+Let's dive a bit deeper into the problem.
+
+**There are many people that don't think coding is for them.**
```

Comment:
Although I think similar messaging would be effective, I'm hesitant of the
specific example of "10 years ago you had to be a genius to code" because
I'm not sure if today's coding landscape is different enough than it was 10
years ago to create change that large.

Sure, learning to code now is easier than it's ever been, but people have
learned to code for decades without being geniuses.

On Mon, Aug 31, 2015 at 10:55 AM, David Fontenot <notifications@github.com>
wrote:

> In first_meeting.md
> <https://github.com/hackedu/hackedu/pull/121#discussion_r38338784>:
>
> > +```
> > +
> > +### Experiencing success on the first day
> > +
> > +Everyone should ship something on their first day at Hack Club. It's the job of
> > +anyone else at the club who's been there at least once to help any newcommers
> > +ship that website.
> > +
> > +This being the first day of club this year, everyone in the entire club
> > +should ship a new website, even if they shipped one previously.
> > +
> > +**Why is it so important to ship something on someone's first meeting?**
> > +
> > +Let's dive a bit deeper into the problem.
> > +
> > +**There are many people that don't think coding is for them.**
>
> One technique I would use is to agree that 10 years you had to be a genius
> to code, but then go on to let them know that the conditions had changed
> and now anyone could do it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hackedu/pull/121/files#r38338784>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us